src/CbcSolverInterface.jl

```diff
@@ -35,7 +42,20 @@ end
 mutable struct CbcSolver <: AbstractMathProgSolver
     options 
 end
-CbcSolver(;kwargs...) = CbcSolver(kwargs)
+
+if VERSION < v"0.7.0-DEV.2738"
+    CbcSolver(;kwargs...) = CbcSolver(kwargs)
+else
+    # Convert keyword arguments to the representation in Julia 0.6,
+    # for backwards compatibility.
```

Comment:
Deprecation fixes should be forward looking so that we can easily remove the code for Julia 0.6 when the time comes. Anyway doesn't the definition below work for both 0.6 and 0.7 and not require any comments?